### DISCUSSION TITLE
Nuxt v3 : project generation, missing feature ?

### DISCUSSION THREAD

**Discussion by Tchoupinax** [nuxt/nuxt#18135] [General]
Hello,

I follow Nuxt v3 since several months and I made some test all along its evolution. I really love Nuxt framework for its features but mainly for its structure. In one minute with Nuxt v2, I am able to create a Vue project with router, Tailwindcss integration, eslint and formatter, axios, PWA, etc..

I am very suprising that Nuxt v3 does not offer this feature and that the generated structure is very poor. The CLI does not ask for what you want and the nuxt config is blank (the skeleton of the config is very useful and an important expectation for a framework).

I was looking for an issue or a discussion about this but I did not find. I am wondering if this missing and important feature is not here because of the *open beta* status of this project and once this project will be production ready, this feature will come back.

Is it right ?

Thank you for your work :)

**Comment by wobsoriano** [comment]
https://github.com/nuxt/framework/discussions/1029
